Find a functionally equivalent code:android.provider.Settings.Global.getStringForUser:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -1,3 +1,7 @@
 {
+    if (MOVED_TO_SECURE.contains(name)) {
+        Log.w(TAG, "Setting " + name + " has moved from android.provider.Settings.Global" + " to android.provider.Settings.Secure, returning read-only value.");
+        return Secure.getStringForUser(resolver, name, userHandle);
+    }
     return sNameValueCache.getStringForUser(resolver, name, userHandle);
 }

